src/application/services/useNote.ts

```diff
@@ -265,6 +278,35 @@ export default function (options: UseNoteComposableOptions): UseNoteComposableSt
     parentNote.value = undefined;
   }
 
+  /**
+   * Reform the array of note parents by adding the actual note id and content.
+   * @returns An array of Note objects representing the formatted note parents.
+   * @throws {Error} If the note id is not defined.
+   */
+  function formatNoteParents(): Note[] {
+    if (currentId.value === null) {
+      return [];
+    }
+    let presentationFormat: Note[] = [];
+
+    if (noteParents.value.length === 0) {
+      presentationFormat.push({
+        id: currentId.value,
+        content: note.value?.content as NoteContent,
+      });
```

Comment:
why you are adding current note as a parent? I don't think that it is a good idea